README.md

```diff
@@ -1 +1,69 @@
-# My-Portfolio-Project
\ No newline at end of file
+📖 MY PORTFOLIO PROJECT
+This is My Portfolio Project dashboard.
+
+My Portfolio Project dashboard my skills as a software developer.
+
+🛠 Built With
+Tech Stack
+
+Client
+HTML and CSS
+
+🚀 Live Demo
+Add a link to your deployed project.
+
+https://github.com/AbednegoKibitok/My-Portfolio-Project
+
+💻 Getting Started
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need: HTML, CSS, Code editor and a Browser.
+
+Setup
+Clone this repository to your desired folder: https://github.com/AbednegoKibitok/My-Portfolio-Project
+
+Deployment
+You can deploy this project using: Visual Studio Code
+
+👥 Authors
+Mention all of the collaborators of this project.
+
+👤 Author1
+
+GitHub: @AbednegoKibitok
+Twitter: @AbednegoKibitok
+LinkedIn: Abednego Kibitok
+
+🔭 Future Features
+Describe 1 - 3 features you will add to the project.
+
+My projects
+Languages that I am proficient in.
+My frameworks
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+(back to top)
+
+⭐️ Show your support
+Write a message to encourage readers to support your project
+
+If you like this project, contribute to it via my github page.
+
+(back to top)
+
+🙏 Acknowledgments
+Give credit to everyone who inspired your codebase.
+
+I would like to thank Microverse community for such a great support.
+
+(back to top)
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Please make use of the [new README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided and make your edits using [Markdown syntax.](https://www.markdownguide.org/basic-syntax/). 

**Your README File**

![image](https://user-images.githubusercontent.com/84156714/212358413-8b249527-b212-4d3e-8dbb-1fa6c3aba186.png)

**Microverse README Template**

![image](https://user-images.githubusercontent.com/84156714/212358529-da5cf730-6750-46d1-802f-d6182ecbe0a7.png)

![image](https://user-images.githubusercontent.com/84156714/212358588-0e7873a4-2557-46b2-8025-040471ecded5.png)